src/js/components/sharedComponents/checkbox/PrimaryCheckboxType.jsx

```diff
@@ -61,8 +63,8 @@ export default class PrimaryCheckboxType extends React.Component {
         super(props);
 
         this.state = {
-            showSubItems: false,
-            arrowState: 'collapsed',
+            showSubItems: !props.collapsable,
```

Comment:
I understand this to be an antipattern in react... we might be doing it in places; but we likely should never do this. If we're just porting something from props to state directly; it should remain props.

Reply:
This is especially confusing as were setting a positive attribute `show...` w/ a falsy value. Would recommend making this positive. ie:

`hideSubItems: collapsable` 

That said, this should be props rather than derived state from props.

Reply:
Made a change to handle this.